src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -599,6 +599,15 @@ public String getStatus() {
         return statusMessage == null ? "" : statusMessage.getText();
     }
 
+    public String[] getStatuses() {
+        List<WebElement> statuses = browser.driver.findElements(By.cssSelector("#statusMessagesToUser > div"));
```

Comment:
Don't use `cssSelector`. What if the id changes, you have to change it in two locations.

Reply:
The reason I used `cssSelector` is without it (if I use `By.id("statusMessagesToUser"`)) the status messages get concatenated automatically giving rise to the same problem we want to avoid.

Reply:
There isn't just one way of doing things, look harder.

Reply:
Okay. (Sorry I am new to selenium :sweat_smile:)

Reply:
@LiHaoTan Can you `xpath` in place of `cssSelector`? <br>This seems to work : `By.xpath(".//*[@id='statusMessagesToUser']/div")`.

Reply:
Of course you can, but my point is more of the magic string `statusMessagesToUser`, not the method you are calling. By the way, `cssSelector` is preferable to `xpath`.